library/include/rocwmma/internal/types.hpp

```diff
@@ -35,11 +35,7 @@
 #include <utility>
 #endif // !__HIPCC_RTC__
 
-#if defined(__HIPCC_RTC__)
-#define __HOST_DEVICE__ __device__
-#else
-#define __HOST_DEVICE__ __host__ __device__
-#endif
+#include <rocwmma/internal/config.hpp>
```

Comment:
#include "config.hpp" as the current file is required to be colocated in the same folder.
Using the <> brackets include nomenclature usually refers to external library includes not for the current project.